pagerduty/event_orchestration_path.go

```diff
@@ -170,6 +176,7 @@ func (s *EventOrchestrationPathService) Update(id string, pathType string, orche
 func (s *EventOrchestrationPathService) UpdateContext(ctx context.Context, id string, pathType string, orchestrationPath *EventOrchestrationPath) (*EventOrchestrationPathPayload, *Response, error) {
 	u := orchestrationPathUrlBuilder(id, pathType)
 	v := new(EventOrchestrationPathPayload)
+	sanitizeOrchestrationPath(orchestrationPath)
```

Comment:
Thank you for fixing this! Could you also add a test verifying that if you pass all of these properties as `nil` the payload is sent to the API with empty arrays instead? The test can be added for any orchestration type but the real world scenario would be a service orchestration since it supports all of these actions.

Reply:
Thanks. I just added some tests around this and had to address a previous test.

Reply:
ty!
My go is a little rusty rusty but I'll give a lgtm because nothing is blocking

Reply:
Hey @swbradshaw thank you for introducing this fix and because it is a patch, I'd prefer to move this patch to another PR in the current one can depend on before it gets merged. On top of that, being that this PR is not planned to be merged soon, because it relies on having its corresponding API Reference updated too, then We would be able to ship this bugfix first. So, could you please move this fix to another PR?

Reply:
I don't have time to create a new PR for this. Given this library can't create orchestrations without supplemental code and no one has complained, I don't think a patch needs to be merged sooner than this.  If you don't want it here, I'll just remove all these changes related to sanitizing the request.